internal/cli/commands.go

```diff
@@ -182,3 +182,26 @@ func versionCmd(m *migrate.Migrate) {
 		log.Println(v)
 	}
 }
+
+func numDownMigrationsFromArgs(applyAll bool, args []string) (int, bool, error) {
```

Comment:
Document the return values since it's unclear what the `bool` is for without seeing example usage

Reply:
added some comments!